### ISSUE TITLE
Add dark mode to the site

### ISSUE THREAD

**Issue by Himanshu-Agg12** [json-schema-org/website#130]
Add a button in the header for dark mode. The dark mode is more readable for the user.
Please assign this task to me i wanted to contribute to this great project.

**Comment by benjagm** [comment]
Hi @Himanshu-Agg12 . Are you still available to implement this enhancement?

**Comment by Min2who** [comment]
Can i get assigned as it is fairly a big issue !

**Comment by praveen-rikhari** [comment]
Hey @benjagm any suggestions on how should we go ahead with this?? Do we have a theme for dark mode that we can leverage upon??

**Comment by benjagm** [comment]
> Do we have a theme for dark mode that we can leverage upon??

We don't have any theme to use. We need to build it from scratch.

**Comment by benjagm** [comment]
@Min2who @praveen-rikhari the issue is big, maybe you can consider working together on it. :-)

**Comment by praveen-rikhari** [comment]
Sure @benjagm, @Min2who how would you like to proceed on this??

**Comment by Min2who** [comment]
> Sure @benjagm, @Min2who how would you like to proceed on this??

I will make a draft PR soon You can commit in it ? Is this okay

**Comment by benjagm** [comment]
If you are still working on the PR please update it to draft status until is ready to review. Great progress here!

**Comment by benjagm** [comment]
If you are still working on the PR please update it to draft status until is ready to review. Great progress here!

**Comment by benjagm** [comment]
@praveen-rikhari please continue the great work done by @utnim2. He did a lot of progress in the PR #365 
Let's make sure you continue the work from there.

**Comment by praveen-rikhari** [comment]
@benjagm currently working on these two issues 
- #181
- #144
So, please can you unassign me and leave this for other contributor & @utnim2 Great work so far looks very good.
Thank you

**Comment by praveen-rikhari** [comment]
@benjagm currently working on these two issues
- https://github.com/json-schema-org/website/issues/181
- https://github.com/json-schema-org/website/issues/144

So, please can you unassign me and leave this for other contributor & @utnim2 Great work so far looks very good.
Thank you

**Comment by benjagm** [comment]
Ok @praveen-rikhari . @utnim2 Are you ok with finalising alone?

**Comment by praveen-rikhari** [comment]
@benjagm Thanks

**Comment by praveen-rikhari** [comment]
@DhairyaMajmudar Sounds great, please ask @benjagm @utnim2  regarding this.

**Comment by utnim2** [comment]
@DhairyaMajmudar you have 7 issues assigned it to yourself 
![image](https://github.com/json-schema-org/website/assets/127925465/dc798cd2-d4fc-41a4-93eb-8a7c8c5dea47)
It will be nice if a new contributor wants to do the work

**Comment by DhairyaMajmudar** [comment]
Yes I have completed these issues and also linked a PR all of them : )

I think it will would be good issue for me.

**Comment by benjagm** [comment]
Thanks @DhairyaMajmudar for offering support . I have discussed with @utnim2 and we think is ok for him to complete the PR.

**Comment by benjagm** [comment]
@jayprakash25 are you interested in continuing the great work done by @utnim2 for this issue in the PR https://github.com/json-schema-org/website/pull/365?

**Comment by benjagm** [comment]
> Yes I have completed these issues and also linked a PR all of them : )
> 
> I think it will would be good issue for me.

Everyone is doing a great job!

**Comment by jayprakash25** [comment]
Yes @benjagm , I would like to contribute for this issue , may I know what else is to be done.

**Comment by benjagm** [comment]
All yours @jayprakash25 . Please sync with @utnim2 and continue with the draft PR he progressed.

**Comment by jayprakash25** [comment]
Thanks @benjagm .
